src/internal/deploy/cmds/cmds.go

```diff
@@ -557,8 +557,10 @@ func standardDeployCmds() []*cobra.Command {
 				opts.ExposeObjectAPI = true
 
 				// Set the postgres and etcd nodeports explicitly for developers
-				opts.PostgresOpts.Port = 32228
-				opts.EtcdOpts.Port = 32379
+				if !enterpriseServer {
+					opts.PostgresOpts.Port = 32228
```

Comment:
Aren't these set elsewhere? I think they should be in constants.

Reply:
I switched the postgres port to `dbutil.DefaultPort`. In general our story around what connects to what in tests, etc. seems pretty chaotic - we seemingly have a bunch of scripts to port-forward postgres, run it in a docker container, and we run it with a nodeport so it's accessible at the minikube IP. More than reducing the use of constants in the codebase I would love to have a more consistent development flow, which might just eliminate these nodeports altogether.

Reply:
I agree that we could parameterize some of the dbutil stuff so that you can better configure the test environment, Maybe through environment variables.  It would be nice if it wasn't hard coded to use the local forwarded port.

Aren't there 2 Postgres instances though?  They can't both be on the default port if we want to forward them both?

Reply:
This is inside a `!enterpriseServer` block - it's only set for pachd. We have no reliance on the nodeport for the enterprise server tests, so we don't care what port gets assigned.